ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/Address.java

```diff
@@ -176,6 +176,10 @@ public static Address privateContractAddress(
                 })));
   }
 
+  public static Address fromPlugin(final org.hyperledger.besu.plugin.data.Address logger) {
```

Comment:
Should we do a type check first?  There's a strong chance it is a compatible object we just can't staticly prove it.
``
if (logger instanceof Address) {
  return (Address) logger;
} else {
  return wrap(BytesValue.fromPlugin(logger));
}
```